javascripts/DOM.js

```diff
@@ -39,3 +39,73 @@ var Chatty = (function(Chatty) {
 })(Chatty || {});
 
 console.log("DOM js", Chatty);
+
+ document.getElementById("input").addEventListener("keypess", function(event) {
+ // console.log("event",event );
+var textInput = document.getElementById("input").value;
+var outputEl = document.getElementById("container");
+    var string = "";
+    var counter = 1;
+  if (event.keyCode === 13) {
+    outputEl.innerHTML = event.target.value;
```

Comment:
If someone presses enter within the text input field, we just need to call the function Chatty.addNewMessage and pass the input as an argument. The rest of our code kicks in from there once it's all connected. We also don't want `var textInput = document.getElementById("input").value;` because we want to still see the old messages when we add a new one.